test/compiler/EscapeAnalysis/interprocedural.jl

```diff
@@ -23,11 +23,15 @@ end
 
 # MethodMatchInfo -- global cache
 let result = code_escapes((SafeRef{String},); optimize=false) do x
+        # TODO: This is to prevent this function from having ConstABI,
+        # which currently breaks EA
+        Base.donotdelete(1)
         return noescape(x)
     end
     @test has_no_escape(ignore_argescape(result.state[Argument(2)]))
 end
 let result = code_escapes((SafeRef{String},); optimize=false) do x
+        Base.donotdelete(1) # TODO
```

Comment:
could you add a xref to these todo, either copied from above or with something like `TODO: see #ConstABI_issue`